src/main/webapp/mentors.js

```diff
@@ -79,7 +140,66 @@ function createMentorElement(mentor) {
 }
 
 /**
- * Creates a github profile button element from username.
+ * Creates an email for mentor card.
```

Comment:
Nit: fix this in all of the comments that mention mentor card.
```suggestion
 * Creates an email for a mentor card.
```